c7n/resources/account.py

```diff
@@ -1243,3 +1243,72 @@ def process(self, resources):
             client.put_public_access_block(
                 AccountId=r['account_id'],
                 PublicAccessBlockConfiguration=config)
+
+
+@actions.register('set-glue-encryption')
+class SetGlueEncryption(BaseAction):
+    """Set Glue encryption settings (dataCatalog and connection pws)
+    :example:
+    .. code-block:: yaml
```

Comment:
fyi doc building is failing here, you need to include a new line before the code block 

more info here: http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

Reply:
@thisisshi : i have added a new line although it show me same message. Is there a way to check this on my end?